### DISCUSSION TITLE
Das Grundschul Lexikon A

### DISCUSSION THREAD

**Discussion by GoogleNLPAPI** [Estheryu991/SwissGerman_Dictionary#78] [General]
> Aal, der 

> Aale sind Fische. Die Weibchen werden bis zu 150cm lang, die Männchen nur etwa 60cm. Zehn Jahre lang leben sie in Flüssen und Seen, auch bei uns in Europa. Dann schwimmen Sie etwas 6000km durch den Atlantischen Ozean, um sich vor der Küste Amerikas fortzupflanzen - dort, wo sie selbst auch geboren wurden. Die frisch geschlüpften Larven brauchen drei Jahre, um sich von der Meeresströmung bis nach Europa treiben zu lassen.

**Comment by GoogleNLPAPI** [comment]
> Zürich 
> 
> Aal, der
> 
> Aal sind Fisch. D Wiibli werdet bis zu 150cm läng, d Männli nur öbe 60cm. Zäh Jahr lebed sie i Flüss und Seen, au bi üs in Europa. Dänn schwimmed Sie öbe 6000km dur de Atlantische Ozean, um sich vor de Küste Amerikas fortzpflanze - dette, wo sie selber au gebore worde sind. Die frisch gschlüpfte Larve bruchet drü Jahr, um sich vo de Meereströmig bis nach Europa triibe zlah.

**Reply by AalderWaal** [reply]
![google-cloud-natural-language-api](https://user-images.githubusercontent.com/118073998/201489240-d0af6f6f-0bfa-40dc-b7a8-b47fdeb70260.png)

**Reply by AalderWaal** [reply]
Aal Audio DE:


https://user-images.githubusercontent.com/118073998/201489257-2660b658-f743-43ec-a0c6-5acab9ca9940.mp4

**Reply by AalderWaal** [reply]
Aal Audio ZRDE:


https://user-images.githubusercontent.com/118073998/201489272-74707089-8283-4354-83f5-041f107171e6.mp4

**Comment by GoogleNLPAPI** [comment]
> Aas: das 
Tote Tierkörper, die nicht sofort gefressen oder verwertet werden, sondern verwesen, also von kleinen Lebewesen zersetzt werden, nennen wir Aas. Manche Tiere fressen Aas, z. B. Geier, Ratten oder Würmer.

**Reply by AalderWaal** [reply]
Zürich
Aas: das
Toti Tierkörper, die ned sofort gfrässe oder verwertet werded, sondern verweset, als vo chline Lebewese zersetzt werdet, nennen wir Aas. Mängi Tier frässed Aas, z.B. Geier, Ratte oder Würm.

**Reply by AalderWaal** [reply]
Aas DE:


https://user-images.githubusercontent.com/118073998/201489309-8ad11779-60ae-40aa-807a-21ad924b2b5d.mp4

**Reply by AalderWaal** [reply]
Aas ZR DE:


https://user-images.githubusercontent.com/118073998/201489328-de0bb8c0-c689-402f-841e-55cc25bf3f44.mp4

**Comment by AalderWaal** [comment]
Abgase, die

Die Abgase sind gasförmige Abfallstoffe. Sie entstehen bei Verbrennungen, z.B. in Fabriken oder in Automotoren, die mit Benzin oder Diesel betrieben werden. Werden Abgase ungereinigt ausgestossen, verschmutzen sie die Luft und können Menschen, Tieren und Pflanzen schaden.

**Reply by AalderWaal** [reply]
Abgase, die

Abgas sind gasförmigi Abfallstoff. Sie entstenhd bi Verbrennige, z.B. i Fabrike oder in Automotore, die mit Benzin oder Diesel betriebet werdet. Werdet Abgas ungereignit usgstosse, verschmutzet sie d Luft und könnt Mensche, Tiere oder Pflanze schade.

**Reply by AalderWaal** [reply]
Deutsch:
https://user-images.githubusercontent.com/118073998/201494155-00d0145e-8c3d-414c-b6bc-a3ed1d1e204a.mp4

Zürich Deutsch:
https://user-images.githubusercontent.com/118073998/201494158-a92ab067-c8b8-4d4c-89b6-f3db81c74d13.mp4

**Reply by AalderWaal** [reply]
https://user-images.githubusercontent.com/118073998/201494180-c60932a6-ddc0-4688-8bfa-b404af83d970.mp4

**Reply by AalderWaal** [reply]
https://user-images.githubusercontent.com/118073998/201494189-f6b36f6f-721e-44b6-98eb-208e928f4aa5.mp4

**Comment by GoogleNLPAPI** [comment]
> Aborigines, die 

> Eine Bezeichnung für die verschiedenen indiegenen Gruppen Australiens ist "Aborigines". Sie selbst nenne sich "indigenous" Australiens.

**Comment by GoogleNLPAPI** [comment]
> Aborigines, d
> 
> E Bezeichnig für di verschiedene indigene Gruppe vo Australie isch "Aboriginies". Sie selber nenned sich "indigenous" Australiens.

**Reply by esthicodes** [reply]
Vor etwa 40,000 bis 60,000 Jahren begannen Menschen wahrscheinlich vom heutigen Indonesien aus mit der Besiedelung Australiens. Jahrtausendelang lebten sie in kleinen Gruppen zusammen. Im Jahre 1780 wurde die Ostküste Australiens für die britische Krone "in Besitz genommen"(kolonisiert), ab 1788 kamen immer mehr Menschen aus Europa, um in Australien zu leben oder dort als Strafgefangene gehalten zu werden. Die indigenen Menschen wurden meist unterdrückt und vertrieben. (Sie müssen weg)

> Viele von Ihnen starben durch Krankheiten, die die Siedler und Siedlerinnen einschleppten, z.B. Grippe oder Pocken. Erst in den 1960er-Jahren wurden indigene Australierinnen und Australier offiziell [gleichberechtigt(rechtlich gleichgestellt; mit gleichen Rechten ausgestattet)](https://de.wikipedia.org/wiki/Gleichberechtigung).

![image](https://user-images.githubusercontent.com/78131082/201883880-e48e76e9-9784-42a5-8773-d59b104c042b.png)


[Source](https://www.domradio.de/system/files/styles/w21_dmr_theme_full_xl_2x/private/aborigines-sydney-bei-der-traditionellen-smoking-ceremony-anlaesslich-des-australia-day.jpg?h=ae1281eb&itok=ELMxjhyT)

**Reply by esthicodes** [reply]
<BERN>
Vor öppä 40,000(vierzgtusig) bis 60,000(sechzgtusig) Jahr hei Mönschä wahrschinlech vom hütigä Indonesiä us mitder Besiedlig vo Oustraliä agfangä.

Jahrtusigläng heisi i chline Gruppe zämegläbt. Im Jahr 1780 isch t Ostküstä vo Oustralie für di britischi Chrone "in Besitz gno" wordä (kolonisiert), ab 1788 si immer meh Mönschä us Europa cho, um in Oustralä z läbe oder dört als Strafgefangeni ghautä zwärde. Di indigenä Mönschä si meist ungerdrückt und vertriebä wordä. (Sie müessä wäg)

Vieli vo ihnä si dür Chrankheitä gstorbä, wo t Siedler und Siedlerinnö iigschleppt hei, z.B. Grippe oder Pockä. Ersch idä 1960er-Jahr si di indigenä Oustralierinne und Oustralier offiziell anerkennt wordä.[gleichberechtigt(rechtlich gleichgestellt; mit gleichen Rechten ausgestattet)](https://de.wikipedia.org/wiki/Gleichberechtigung).

**Comment by cyrillknecht** [comment]
Automat, der

En Automat isch e Maschine, wo e Arbet verrichtet, ohni dass en Mensch debi si muess. En Automat chan recht eifachi wiederkehrendi Ufgabe usfüere, wie zum Bispil en Getränkeautomat , oder kompliziert ufbaut si, wie zum Bispil Roboter, wo Autoteil zämesetze chönd.

**Comment by esthicodes** [comment]
Abfahrt:
_die Bus habe abfahrt gemacht 
Der Bus ist schon Abfahrt gegangen_
Der Bus ist schon abgefahren

Abgas:
ich hasse die Abgas von China weil sie gefärdet die Umwelt. 

Auspuff: 
aus dem Auspuff kommt die Abgas.
